src/components/messenger/list/index.tsx

```diff
@@ -111,11 +122,17 @@ export class Container extends React.Component<Properties, State> {
       back,
       startGroup,
       membersSelected,
+      fetchRewards,
       enterFullScreenMessenger: () => enterFullScreenMessenger(),
     };
   }
 
-  state = { isRewardsPopupOpen: false, isRewardsFAQModalOpen: false };
+  state = {
+    isRewardsPopupOpen: false,
+    isRewardsFAQModalOpen: false,
+    isRewardsTooltipOpen: true, // initally open, will close after user clicks on 'x' button
```

Comment:
This is going to be open every time you log in? Shouldn't that only happen if you have new rewards? Also, the very first time you register we're supposed to have the whole rewards popup open. Does this cause the tooltip and the modal to be open at the same time?

Reply:
done, thanks